pkg/iris/api.go

```diff
@@ -70,3 +71,30 @@ func (a *api) PostResults(results []byte, accessToken string) error {
 	log.Infof("Event posted successfully [%s]", resp.Status)
 	return nil
 }
+
+func (a *api) PostStatus(status []byte, accessToken string) error {
+	resultUrl := fmt.Sprintf("https://%s/services/vsm-iris/v1/status", a.uri)
```

Comment:
what is the difference between `/status` & `/results`

Reply:
Endpoints `/status` and `/results` have different schema objects and the respective target eventbus are also different such as 'discovery_items' and 'feedback_items'.